api/lib/opentelemetry/instrumentation/base_instrumentation.rb

```diff
@@ -225,12 +225,12 @@ def enabled?(config = nil)
 
       private
 
-      # Checks to see if this adapter is enabled by env var. By convention, the
-      # environment variable will be the adapter name upper cased, with '::'
+      # Checks to see if this instrumentation is enabled by env var. By convention, the
+      # environment variable will be the instrumentation name upper cased, with '::'
       # replaced by underscores and _ENABLED appended. For example, the
-      # environment variable name for OpenTelemetry::Adapter::Sinatra will be
-      # OPENTELEMETRY_ADAPTERS_SINATRA_ENABLED. A value of 'false' will disable
-      # the adapter, all other values will enable it.
+      # environment variable name for OpenTelemetry::Instrumentation::Sinatra will be
+      # OPENTELEMETRY_INSTRUMENTATIONS_SINATRA_ENABLED. A value of 'false' will disable
```

Comment:
The comment has `INSTRUMENTATIONS`. As an FYI, there is spec [PR #666](https://github.com/open-telemetry/opentelemetry-specification/pull/666) that recomends forming environment language specific environment variables as `OTEL_{LANGUAGE}_{FEATURE}`. Which, would mean `OTEL_RUBY_INSTRUMENTATION_SINATRA_ENABLED`. We could consider making this change now so we only have to make it once, or track it separately.

Reply:
cursed spec number. Anyway, I've updated this to `OTEL_RUBY_INSTRUMENTATION_SINATRA_ENABLED` but, i'll admit, my approach is not the cleanest. I think it's a good approach, i'd be slightly concerned about this change (and all the others here) breaking things for folks using this library currently? Or, because this is in alpha, that's assumed that things may break from version to version?